qemu/tests/cfg/fio_perf.cfg

```diff
@@ -0,0 +1,57 @@
+- fio:
+    only x86_64
+    type = fio_perf
+    numa_node = -1
+    block_size = "4k 16k 64k 256k"
+    format = False
+    iodepth = "1 8 64"
+    threads = "16"
+    rw = "read write randread randwrite randrw"
+    Linux:
+        no ide
+        pattern = ".*[read|write].*bw=(\d+(?:\.\d+)?[\w|\s]B/s),\siops=(\d+).*"
+        order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits Util%"
+        guest_ver_cmd = "uname -r"
+        compile_cmd = "make && make install"
+        pre_cmd = "i=`/bin/ls /dev/[vs]db` && mkfs.xfs $i > /dev/null; partprobe; umount /mnt; mount $i /mnt"
+        drop_cache = "sync && echo 3 > /proc/sys/vm/drop_caches"
+        guest_result_file = /tmp/fio_result
+        fio_cmd = "fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/mnt/%s --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output=/tmp/fio_result &> /dev/null"
+    Windows:
+        guest_ver_cmd = wmic datafile where name="c:\\windows\\system32\\drivers\\viostor.sys" || wmic datafile where name="c:\\windows\\system32\\drivers\\vioscsi.sys"
+        pattern = ".*?\s{2}[read|write].*?bw=(\d+(?:\.\d+)?[\w|\s]B/s),\siops=(\d+)"
+        order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits"
+        pre_cmd = "echo select disk 1 > imDiskpart.script && echo create partition primary >> imDiskpart.script && echo assign letter=I >> imDiskpart.script&& echo exit >> imDiskpart.script && diskpart /s imDiskpart.script"
+        online_disk_cmd = "echo select disk %s > imDiskpart.script && echo online disk >> imDiskpart.script && echo attr disk clear readonly >> imDiskpart.script && echo exit >> imDiskpart.script && diskpart /s imDiskpart.script"
+        guest_result_file = "C:\\fio_result"
+        fio_cmd = 'cmd /c C:\fio-2.0.15-x64\fio.exe --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=I:\%s --name=job1 --ioengine=windowsaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output="C:\\fio_result"'
+    variants:
+        - localfs:
+            image_name_disk1 = /mnt/storage2
+            Linux:
+                vhost_nic1 =
```

Comment:
For performance, we don't need to test vhost=on on localfs backend, this is our test case,  so I set vhost_nic1=null. For nfs, we need to test vhost=on, it is correct.